### DISCUSSION TITLE
How can I remove the Google Map and Directions buttons at the bottom right?

### DISCUSSION THREAD

**Discussion by parkjunseo1** [react-native-maps/react-native-maps#4826] [Q&A] [answered]
I apologize for my limited English proficiency. 

When I press the <Marker> in <MapView>, 
Google Map and Directions buttons appear at the bottom right, but we don't need them. How can we remove them?

![image](https://github.com/react-native-maps/react-native-maps/assets/51080793/a224e106-505a-47df-abc8-28b0664a9319)

**Comment by arnaldocrescente** [comment, marked as answer]
You can set the MapView props toolbarEnabled to false

**Reply by parkjunseo1** [reply]
Oh! I just realized that what's written in the document. Thank you for your kindness. :)
